packages/core/core/src/AssetGraph.js

```diff
@@ -131,9 +131,10 @@ export default class AssetGraph extends Graph {
 
     for (let assetNode of assetNodes) {
       // TODO: dep should already have sourcePath
-      let depNodes = assetNode.value.dependencies.map(dep =>
-        nodeFromDep({...dep, sourcePath: file.filePath})
-      );
+      let depNodes = assetNode.value.dependencies.map(dep => {
+        dep.sourcePath = file.filePath;
```

Comment:
To discuss: do we even need this field on a dependency? An Asset node points to a Dependency in the graph, so we should be able to infer the file the dep came from based on its location in the graph.

Reply:
Yeah I saw that. But we could pass the source path as a second argument to `nodeFromDep` to create the id... Perhaps it is useful information to store on a dep. I'm not sure.